test/e2e/user_workload_monitoring_test.go

```diff
@@ -76,10 +75,62 @@ func TestUserWorkloadMonitoringInvalidConfig(t *testing.T) {
 	f.MustCreateOrUpdateConfigMap(t, cm)
 	defer f.MustDeleteConfigMap(t, cm)
 
+	t.Log("invalid UWM configuration with malformed YAML/JSON")
 	f.AssertOperatorCondition(configv1.OperatorDegraded, configv1.ConditionTrue)(t)
 	f.AssertOperatorCondition(configv1.OperatorAvailable, configv1.ConditionFalse)(t)
 	f.AssertOperatorConditionReason(configv1.OperatorDegraded, "UserWorkloadInvalidConfiguration")
 	f.AssertOperatorConditionReason(configv1.OperatorAvailable, "UserWorkloadInvalidConfiguration")
+	// Even when an invalid configuration is caught by both unmarshallers, the operator is still set
```

Comment:
no longer needed, we just have one unmarshaller now and it only yields errors, no need to have warnings for it. see above.